pkg/controller/networkpolicy/clusternetworkpolicy_test.go

```diff
@@ -1405,739 +1477,62 @@ func TestProcessClusterNetworkPolicy(t *testing.T) {
 }
 
 func TestAddCNP(t *testing.T) {
-	p10 := float64(10)
-	emergencyTierPriority := int32(1)
-	emergencyTier := crdv1alpha1.Tier{
-		ObjectMeta: metav1.ObjectMeta{Name: "emergency", UID: "tE"},
-		Spec: crdv1alpha1.TierSpec{
-			Priority: emergencyTierPriority,
-		},
-	}
-	appTier := crdv1alpha1.Tier{
-		ObjectMeta: metav1.ObjectMeta{Name: "application", UID: "tA"},
-		Spec: crdv1alpha1.TierSpec{
-			Priority: DefaultTierPriority,
-		},
-	}
-	allowAction := crdv1alpha1.RuleActionAllow
-	protocolTCP := controlplane.ProtocolTCP
-	selectorA := metav1.LabelSelector{MatchLabels: map[string]string{"foo1": "bar1"}}
-	selectorB := metav1.LabelSelector{MatchLabels: map[string]string{"foo2": "bar2"}}
-	selectorC := metav1.LabelSelector{MatchLabels: map[string]string{"foo3": "bar3"}}
-	selectorAll := metav1.LabelSelector{}
-	matchAllPeerEgress := matchAllPeer
-	matchAllPeerEgress.AddressGroups = []string{getNormalizedUID(antreatypes.NewGroupSelector("", nil, &selectorAll, nil, nil).NormalizedName)}
+	_, npc := newController()
+	cnp := getCNP()
+	npc.addCNP(cnp)
+	require.Equal(t, 1, npc.internalNetworkPolicyQueue.Len())
+	key, done := npc.internalNetworkPolicyQueue.Get()
+	expectedKey := getAntreaClusterNetworkPolicyReference(cnp)
+	assert.Equal(t, expectedKey, key)
+	assert.False(t, done)
+}
+
+func TestUpdateCNP(t *testing.T) {
+	_, npc := newController()
+	cnp := getCNP()
+	newCNP := cnp.DeepCopy()
+	newCNP.Spec.Priority = float64(100)
```

Comment:
could you add a comment explaining why we set the priority to this value here?

Reply:
added